docs/developer_guide.rst

```diff
@@ -34,10 +34,8 @@ To run the tests, we recomment using ``tox`` in a virtual environment::
 
 You can also run individual unit tests with ``lit`` directly::
 
-    pip install lit
-    lit -sv ./tests
-
-However, that requires manually setting up the testing environment (``filecheck``, etc).
+    pip install ".[dev]"
```

Comment:
Should this be done earlier? Or is it that tox will do this for you.

It's been a while but iirc tox is basically "make a virtualenv and run tests" in one go.